apps/assessment_registry/schema.py

```diff
@@ -276,6 +211,80 @@ def resolve_external_link(root, info, **kwargs):
         return render_string_for_graphql(root.external_link)
 
 
+class CNAType(DjangoObjectType, UserResourceMixin, ClientIdMixin):
+    question = graphene.Field(QuestionType, required=True)
+
+    class Meta:
+        model = Answer
+        fields = ("id", "question", "answer",)
+
+
+class IssueType(DjangoObjectType, UserResourceMixin):
+    sub_pillar = graphene.Field(AssessmentRegistrySummarySubPillarTypeEnum, required=False)
+    sub_pillar_display = graphene.String(required=False)
```

Comment:
Let's use EnumDescription here instead of graphene.String. It should work with custom resolve.